test/a.py

```diff
@@ -0,0 +1,7 @@
+from __future__ import annotations
```

Comment:
Would you mind moving these two files one level lower? (e.g. in a `test/postponed_annotations` folder?)

Reply:
Done.